nupic/algorithms/anomaly_likelihood.py

```diff
@@ -134,39 +148,65 @@ def computeLogLikelihood(likelihood):
     return math.log(1.0000000001 - likelihood) / -23.02585084720009
 
 
+  @staticmethod
+  def _calcSkipRecords(numIngested, windowSize, learningPeriod):
+    """Return the value of skipRecords for passing to estimateAnomalyLikelihoods
```

Comment:
This doc string could be more clean. Something like this:

```
"""Return the value of skipRecords for passing to estimateAnomalyLikelihoods.

If windowSize was very large (bigger than the amount of data)
then this could just return `learningPeriod`. But when some values
have fallen out of the historical window of anomaly scores then
we have to take those into account as well so we return the
`learningPeriod` minus the number shifted out.

...
```